frontend/src/app/shared/components/storages/functions/storages.functions.ts

```diff
@@ -38,3 +44,23 @@ export function getIconForMimeType(mimeType?:string):IFileIcon {
 
   return fileIconMappings.default;
 }
+
+export function getIconForStorageType(storageType?:string):string {
+  if (storageType && storageIconMappings[storageType]) {
+    return storageIconMappings[storageType];
+  }
+
+  return storageIconMappings.default;
+}
+
+export function makeFilesCollectionLink(storageLink:IHalResourceLink, location:string|null):IHalResourceLink {
+  let query = '';
+  if (location !== null) {
+    query = `?parent=${location}`;
+  }
```

Comment:
I prefer ternary operators over mutable variables

Reply:
changed to ternary